src/services/scheduler.ts

```diff
@@ -75,7 +77,9 @@ export async function start(dbId: string, dbUser:string) {
           output = typeof output === 'string' ? output : JSON.stringify(output);
           await conn.query(query);
         } catch (e) {
-          const error = logUserErr(e);
+          logUserErr(e);
+          // error must be valid JSON as a string
+          const error = JSON.stringify(e, Object.getOwnPropertyNames(e));
```

Comment:
Does this keep the `message` property?

Reply:
this is what was there before my last PR. so yes